### DISCUSSION TITLE
Powertools glitched?

### DISCUSSION THREAD

**Discussion by drfleetles** [EssentialsX/Essentials#4292] [Q&A] [answered]
All the other players that are allowed to use the command, can use the command but I get an error message.
![image](https://user-images.githubusercontent.com/77469527/124119303-b1b07780-dab5-11eb-84a8-4f9aaf94d2a2.png)

any thoughts?

**Comment by triagonal** [comment, marked as answer]
Update to the latest build at https://essentialsx.net/downloads.html, this issue was fixed in a recent dev build.

**Reply by drfleetles** [reply]
ok
